examples/reagent/src/cljs/todomvc/session.cljs

```diff
@@ -0,0 +1,12 @@
+(ns todomvc.session
+  (:require [reagent.core :as reagent]
+            [alandipert.storage-atom :refer [local-storage]]))
```

Comment:
I haven't used this library before so I could be wrong, but I think the key for the todos will become "todos-reagent" from line 5. I should probably rename the key for the counter from "counter' to "counter-reagent" or something though.